assets/training/finetune_acft_image/environments/acft_image_huggingface/context/requirements.txt

```diff
@@ -0,0 +1,17 @@
+azureml-metrics=={{latest-pypi-version}}
+azureml-evaluate-mlflow=={{latest-pypi-version}}
+azureml-acft-accelerator=={{latest-pypi-version}}
+azureml-acft-common-components[image]==0.0.24
```

Comment:
Should this also be changed to {{latest-pypi-version]} ?

Reply:
same as above